content/client-server-api/_index.md

```diff
@@ -1831,6 +1841,221 @@ the topic to be removed from the room.
 
 {{% http-api spec="client-server" api="redaction" %}}
 
+### Forming relationships between events
+
+{{% changed-in v="1.3" %}}
+
+In some cases it is desirable to logically associate one event's contents with
+another event's contents. For example, when replying to a message, editing an
+event, or simply looking to add context for an event's purpose.
+
+Events are related to each other in a parent/child structure, where any event can
+become a parent by simply having a child event point at it. Parent events do not
+define their children, instead relying on the children to describe their parent.
```

Comment:
yea... I'd kinda like to test this terminology in production: if it becomes too confusing, it'd be a (series of) clarification PRs, which are easier to get through than a stack of MSCs 😇

Reply:
We do use `parent_id` in a few spots in the code, I don't think we use "child" anywhere though.

Reply:
I also asked @turt2live to choose clear names to avoid some of the confusing conversations that @richvdh and I had while reviewing this code!

I don't know if parent/child is the proper thing, I think some code also uses target/source. I would stay away from "related" though since you can end up using it to describe both a parent-child relationship as well as a child-parent relations.